routes/cube_routes.js

```diff
@@ -3160,6 +3165,65 @@ router.post('/api/updatecards/:id', ensureAuth, function(req, res) {
   });
 });
 
+function maybeCards(cube) {
+  const maybe = (cube.maybe || []).filter((card) => card.cardID);
+  return maybe.map((card) => ({ ...card, details: carddb.cardFromId(card.cardID) }));
+}
+
+router.get(
+  '/api/maybe/:id',
+  ensureAuth,
+  util.wrapAsyncApi(async function(req, res) {
+    const cube = await Cube.findOne(build_id_query(req.params.id));
+    if (!cube) {
+      return res.status(404).send({
+        success: 'false',
+        message: 'Cube not found.',
+      });
+    }
+
+    return res.status(200).send({
+      success: 'true',
+      maybe: maybeCards(cube),
+    });
+  }),
+);
+
+router.post(
+  '/api/maybe/:id',
+  ensureAuth,
+  util.wrapAsyncApi(async function(req, res) {
+    const cube = await Cube.findOne(build_id_query(req.params.id));
+    if (!cube) {
+      return res.status(404).send({
+        success: 'false',
+        message: 'Cube not found.',
+      });
+    } else if (!req.user._id.equals(cube.owner)) {
```

Comment:
else not needed, as we return in the previous if block

Reply:
just habit and it lets me put it on the same line